edx_lint/pylint/unittest_assert_check.py

```diff
@@ -0,0 +1,107 @@
+"""Checker for using pytest assertion instead of unittest assertion."""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker, utils
+
+from .common import BASE_ID, check_visitors
+
+
+def register_checkers(linter):
+    """Register checkers."""
+    linter.register_checker(UnittestAssertChecker(linter))
+
+
+@check_visitors
+class UnittestAssertChecker(BaseChecker):
+    """
+    Checks if a unit test assertion is used, Trigger warning to
+    replace it with pytest assertions
+    """
+
+    __implements__ = (IAstroidChecker,)
+
+    name = "unittest-assert-checker"
+
+    UNITTEST_ASSERTS = [
+        "assertTrue",
+        "assertFalse",
+        "assertEqual",
+        "assertEquals",
+        "assertNotEqual",
+        "assertNotEquals",
+        "assert_",
+        "assertIn",
+        "assertNotIn",
+        "assertLess",
+        "assertLessEqual",
+        "assertGreater",
+        "assertGreaterEqual",
+        "assertAlmostEqual",
+        "assertNotAlmostEqual",
+        "assertIs",
+        "assertIsNot",
+        "assertIsNone",
+        "assertIsNotNone",
+        "assertIsInstance",
+        "assertNotIsInstance",
+        "assertRaises",
+    ]
+
+    ASSERT_MAPPING = {
+        "assertEqual": "assert arg1 == arg2",
+        "assertEquals": "assert arg1 == arg2",
+        "assertNotEqual": "assert arg1 != arg2",
+        "assertNotEquals": "assert arg1 != arg2",
+        "assert_": "assert arg1",
+        "assertTrue": "assert arg1",
+        "assertFalse": "assert not arg1",
+        "assertIn": "assert arg1 in arg2",
+        "assertNotIn": "assert arg1 not in arg2",
+        "assertIs": "assert arg1 is arg2",
+        "assertIsNot": "assert arg1 is not arg2",
+        "assertIsNone": "assert arg1 is None",
+        "assertIsNotNone": "assert arg1 is not None",
+        "assertIsInstance": "assert isinstance(arg1, arg2)",
+        "assertNotIsInstance": "assert not isinstance(arg1, arg2)",
+        "assertLess": "assert arg1 < arg2",
+        "assertLessEqual": "assert arg1 <= arg2",
+        "assertGreater": "assert arg1 > arg2",
+        "assertGreaterEqual": "assert arg1 >= arg2",
+        "assertAlmostEqual": "assert round(arg1 - arg2, 7) >= 0",
+        "assertNotAlmostEqual": "assert round(arg1 - arg2, 7) != 0",
```

Comment:
The math module has [isclose](https://docs.python.org/3/library/math.html#math.isclose) which would be a better replacement.

Reply:
Replaced with `isclose`